src/jump.jl

```diff
@@ -8,18 +8,143 @@ Note that if non-linear constraint are present in the model, they are ignored.
 """
 hrep(model::JuMP.Model) = LPHRepresentation(model)
 
+function _get_all_constraint_ref_array(model::Model)
+    ref_arr = vcat([
```

Comment:
Splatting with an arbitarily long vector is not a good idea, use `reduce(vcat, [...])` instead of `vcat([...]...)`

Reply:
Very good point - thanks! Fixed.